packages/web-components/package.json

```
@@ -8,7 +8,7 @@
   "module": "index.js",
   "scripts": {
     "build": "exit 0",
-    "test": "web-test-runner --coverage",
+    "test": "if test \"$SKIP_BROWSER_TESTS\" != \"true\" ; then web-test-runner --coverage ; fi",
```

Is this intended to work on Windows? If so, I think it needs a tweak like e.g.
```suggestion
    "test": "node -e 'process.exit(process.env.SKIP_BROWSER_TESTS==\"true\" ? 0 : 1)' || web-test-runner --coverage",
```
or something more sophisticated such as [run-script-os](https://www.npmjs.com/package/run-script-os).